std/wasi/testdata/std_env_args.rs

```diff
@@ -0,0 +1,6 @@
+// { "args": ["one", "two", "three" ]}
+
+fn main() {
+  let args = std::env::args();
+  assert_eq!(args.len(), 3);
+}
```

Comment:
I wonder if we could build this as part of the deno cargo workspace:

Imagine having std/wasi/testdata/Cargo.toml
```toml
[build]
target = "wasm32-wasi"

[[bin]]
name = "std_env_args"
path = "./std_env_args.rs"

[[bin]]
name = "std_env_vars"
path = "./std_env_vars.rs"
```

Reply:
> [build]
target = "wasm32-wasi"

This doesn't work unfortunately, the default target goes in .cargo/config of the workspace as far as I know, not sure if it can be overriden at any arbritrary level of .cargo files or if its the top level workspace only.

Also by default; the build ends up in `deno/target/<debug|release>` which seems to again be a .cargo/config thing.

Besides that I'm not sure if Cargo manifests are capable of expressing the future scenario of "build these files with that ABI into this folder, and then again with that other ABI into that folder". 

I guess we could *rewrite the build script in Rust* so it runs during cargo build.

```toml
[package]
# ...
build = "build.rs"
```